tests/test_pe_reports.py

```diff
@@ -1,33 +1,43 @@
 """File to be used to create test for pe-reports."""
 
 # Standard Python Libraries
-import unittest
-
-# TODO: Replace current dummy test with useful tests - https://github.com/cisagov/pe-reports/issues/3#issue-909531010
+import sys
+from unittest.mock import patch
 
+# Third-Party Libraries
+import pytest
 
-class TestingStringMethods(unittest.TestCase):
-    """Simple test for first-commit only."""
+# cisagov Libraries
+import pe_reports.report_generator
 
-    def test_string_equality(self):
-        """Calculates the string output."""
-        # if both arguments are equal then it's success
-        self.assertEqual("ttp" * 5, "ttpttpttpttpttp")
+PROJECT_VERSION = pe_reports.__version__
 
-    def test_string_case(self):
-        """Compare two strings."""
-        # if both arguments are equal then it's success
-        self.assertEqual("tutorialspoint".upper(), "TUTORIALSPOINT")
-
-    def test_is_string_upper(self):
-        """Checks whether a string is upper or not."""
-        # used to check whether the statement is True or False
-        # the result of expression inside the **assertTrue** must be True to pass the test case
-        # the result of expression inside the **assertFalse** must be False to pass the test case
-        self.assertTrue("TUTORIALSPOINT".isupper())
-        self.assertFalse("TUTORIALSpoint".isupper())
+# TODO: Replace current dummy test with useful tests - https://github.com/cisagov/pe-reports/issues/3#issue-909531010
 
```

Comment:
Since you have added the `--log-level` option to the main script you should add the [appropriate test from cisagov/skeleton-python-library](https://github.com/cisagov/skeleton-python-library/blob/4ef2364701c0db074c423f740062b2d7c197bea8/tests/test_example.py#L74-L86).

Reply:
Resolved in 253b2dd

Reply:
Resolved in 253b2dd

Reply:
You should include [this test](https://github.com/cisagov/skeleton-python-library/blob/4ef2364701c0db074c423f740062b2d7c197bea8/tests/test_example.py#L89-L93) as well.

Reply:
Added 'bad_log_level' test - Resolved in 25682d6